flow_action_components/GetOpportunityPartners/force-app/main/default/classes/OpportunityPartnersWrapper.cls

```diff
@@ -0,0 +1,45 @@
+/**
+* @author       : Tamar Erlich
+* @date         : October 07, 2020
+* @description  : Wrapper class for opportunity partners
+* Note          : This class is called by the GetOpportunityPartnersAction.
```

Comment:
this is really good documentation. If someone stumbles across this class, you're giving them crucial context.